contracts/root/depositManager/DepositManager.sol

```diff
@@ -88,10 +88,11 @@ contract DepositManager is
     {
         require(_tokens.length == _amountOrTokens.length, "Invalid Input");
         uint256 depositId = rootChain.updateDepositId(_tokens.length);
-
+        Registry _registry = registry;
+        
         for (uint256 i = 0; i < _tokens.length; i++) {
             // will revert if token is not mapped
-            if (registry.isTokenMappedAndIsErc721(_tokens[i])) {
+            if (_registry.isTokenMappedAndIsErc721(_tokens[i])) {
```

Comment:
if this is one time call then we are not saving any gas, maybe extra to keep it in memory.

Reply:
Extra 1 gas for expanding memory by 1 word.